src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -1335,6 +1336,21 @@ default String getOwnerId()
     @Nonnull
     NSFWLevel getNSFWLevel();
 
+    /**
+     * Returns an {@link EnumSet} of {@link SystemChannelFlag} associated to the guild.
+     * <br>For a description of what system channel flags represent, see {@link SystemChannelFlag}.
+     * <p>
+     *
+     * @throws net.dv8tion.jda.api.exceptions.DetachedEntityException
+     *         If this entity is {@link #isDetached() detached}.
+     * @see <a href="https://discord.com/developers/docs/resources/guild#guild-object-system-channel-flags">
+     *     System Channel Flags API documentation
+     *     <a/>
+     * @return the system channel flags of this guild.
+     * */
+    @Nonnull
+    EnumSet<SystemChannelFlag> getSystemChannelFlags();
```

Comment:
Return and document as an **unmodifiable** `Set`, we do not want people to modify the set then complain it doesn't do anything